tests/EnvJsonTest.php

```
@@ -4,56 +4,177 @@
+
+    public function testSchemaWithoutProperties(): void
+    {
+        $result = (new EnvJson())->load(__DIR__ . '/Fake/no-properties-schema');
+        $this->assertInstanceOf(stdClass::class, $result);
+        $this->assertEmpty((array) $result); // Check if the object has no properties
+    }
+
+    public function testUnreadableSchemaFile(): void
+    {
+        // Update expectation based on actual behavior observed in composer coverage
+        $this->expectException(JsonFileNotReadableException::class);
```

PHPStan (2.0.3)

Class Koriym\EnvJson\Exception\JsonFileNotReadableException not found.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(class.notFound)